data-avro/src/main/java/com/linkedin/data/avro/util/AvroUtil.java

```diff
@@ -31,15 +32,40 @@
 {
   public static String jsonFromGenericRecord(GenericRecord record) throws IOException
   {
-    GenericDatumWriter<GenericRecord> writer = new GenericDatumWriter<>();
+    return jsonFromGenericRecord(record, true);
+  }
+
+  public static String jsonFromGenericRecord(GenericRecord record, boolean pretty) throws IOException
+  {
     ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-    Encoder jsonEncoder = AvroCompatibilityHelper.newJsonEncoder(record.getSchema(), outputStream, true);
+    return jsonFromGenericRecord(record,
+        outputStream,
+        AvroCompatibilityHelper.newJsonEncoder(record.getSchema(), outputStream, pretty));
+
+  }
+
+  public static String jsonFromGenericRecord(GenericRecord record, boolean pretty, AvroVersion version) throws IOException
+  {
+    ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+    return jsonFromGenericRecord(record,
+        outputStream,
+        AvroCompatibilityHelper.newJsonEncoder(record.getSchema(), outputStream, pretty,version));
```

Comment:
nit: there should be a space in between "pretty, version".

Reply:
resolved